debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorTest.java

```diff
@@ -6,40 +6,39 @@
 
 package io.debezium.connector.postgresql;
 
-import com.google.common.collect.ImmutableMap;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.StringStartsWith.startsWith;
+
+import java.util.HashMap;
+import java.util.Map;
+
 import org.apache.kafka.common.config.Config;
 import org.apache.kafka.common.config.ConfigValue;
 import org.junit.Before;
 import org.junit.Test;
 
-import java.util.Map;
+public class PostgresConnectorTest {
```

Comment:
Are there any functional changes in this file? AFAICT there's just some shuffling of import and method order.

Reply:
This was a test that depended on guava, which was present in `v1.2.2` before when it was introduced, but removed in `v1.3.1` . I changed the test to remove the use of ImmutableMap to get it to compile. That is all.